CHANGES.rst

```diff
@@ -1,6 +1,9 @@
 0.10.0 (unreleased)
 ===================
 
+- Fix wrong behavior with ``IGNORE_WARNINGS`` and ``SHOW_WARNINGS`` that could
+  make a block to pass instead of being skipped. [GUESSPR]
```

Comment:
Did I win? 😹 
```suggestion
  make a block to pass instead of being skipped. [#148]
```

Reply:
Oups, I don't have the pre-commit hook on this repo ;)